public/editor.js

```diff
@@ -44,11 +44,17 @@ export class Editor
         // List of nodes ids for nodes being dragged
         this.dragNodes = [];
 
-        // Mouse position at the start of a group selection or node movement
-        this.startMousePos = null;
-
-        // Last mouse position during node movement
-        this.lastMousePos = null;
+        // Positional information about the pointer
+        this.pointer = {
+            // Currently known location of the pointer. This is provided by
+            // browser events and manually updated as needed, while
+            // autoscrolling for example.
+            position: null,
+
+            // Location of the pointer when it was last pressed down. Cleared
+            // when the pointer is no longer pressed down.
+            downPosition: null
```

Comment:
I would maybe rename these pos and downPos, otherwise it makes for very long variable names.